aquarius/app/assets.py

```diff
@@ -157,21 +157,21 @@ def register():
             service:
                   type: array
                   description: List of services.
-                  example: [{"type": "Authorization",
+                  example: [{"type": "authorization",
                               "serviceEndpoint": "http://localhost:12001",
                               "service": "SecretStore",
-                              "serviceDefinitionId": "0"
+                              "index": "0"
```

Comment:
all the `index` are integers now https://github.com/oceanprotocol/OEPs/blob/master/8/v0.4/README.md#example-of-local-metadata

Reply:
Thanks @kremalicious , I will update